docs/open-api/openapi.yaml

```diff
@@ -133,6 +133,23 @@ paths:
     $ref: "./roles.yaml#/paths/~1metalakes~1%7Bmetalake%7D~1roles~1%7Brole%7D"
 
   /metalakes/{metalake}/owners/{metadataObjectType}/{metadataObjectFullName}:
+    parameters:
+      - $ref: '#/components/parameters/metalake'
+      - name: metadataObjectType
```

Comment:
can we move the definition to `#/components/parameters`?

Reply:
There is an existing metadataObjectType in #/components/parameters . What should we call this one?
I wrote this here because the issue mentions that only the owner API's metadataObjectType possible values do not include COLUMN.

Reply:
I suggest you update the definition of `owner.yaml` instead of here.
https://github.com/apache/gravitino/blob/9aa8f53f4ca231fe4250e264973785ad17c3f66f/docs/open-api/owners.yaml#L26

Reply:
you can add a parameter `metadataObjectTypeWithoutColumn`, then reference it

Reply:
Ok